### PR TITLE
Add reflection-milestone4.md

### PR DESCRIPTION
**Pull request by Vikiano**

### CODE REVIEW

File: doc/reflection-milestone4.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+> Authors: Arlin Cherian, Victor Francis, Wanying Ye
+
+## 1. What we have implemented
+
+We have made most of the implementations that we outlined in our proposal. These implementations include:
+
+1.  An interactive map of `Crime Density by Neighbourhood, Crime Type, and  Month`;
+
+2.  An interactive bar chart of `Total Reported Cases by Neighbourhood and Crime Type`;
```
Comment (arlincherian) [inline doc/reflection-milestone4.md]: and by crime category

File: doc/reflection-milestone4.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
```
Comment (arlincherian) [inline doc/reflection-milestone4.md:12]: You can also add a line here that says, the improvements made to the py-version of the dashboard is also implemented in R.

Comment (arlincherian) [reply doc/reflection-milestone4.md:12]: Also, we added summary stats for all of Vancouver - you can write about that as well

File: doc/reflection-milestone4.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+The limitations of our dashboard include:
+
+-   Small dataset. The dataset used was for January to December 2021 as released by Vancouver Police Department. It could have been better if we also used crime datasets for preceding years.
+
+-   We did not implement complete reproducibility. That is, for example, this dashboard will not get updated if Vancouver Police Department releases more recent crime data on their website.
+
+## 5. What are potential improvements & additions
+
+If we had more time in the future, we have a couple of potential additions in mind:
+
```
Comment (arlincherian) [inline doc/reflection-milestone4.md:52]: Maybe alsp perfecting the layout views of the containers more, fixing the about section positioning during scrolling

**Review by arlincherian — CHANGES_REQUESTED** [review]
Hey Victor! Thanks for drafting this. I have suggested a few comments, please review and adjust if necessary. thanks!

**Comment by Vikiano** [comment]
Hi Arlin! Thanks for the suggestions; all implemented now.

**Pull request merged**